src/app/rally-finance-list/rally-finance-list.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RallyFinanceListComponent } from './rally-finance-list.component';
+import { MatCardModule } from '@angular/material/card';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { MatIconModule } from '@angular/material';
+import { By } from '@angular/platform-browser';
+
+describe('RallyFinanceListComponentComponent', () => {
+  let component: RallyFinanceListComponent;
+  let fixture: ComponentFixture<RallyFinanceListComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MatCardModule, FlexLayoutModule, MatIconModule],
+      declarations: [RallyFinanceListComponent],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RallyFinanceListComponent);
+    component = fixture.componentInstance;
+    component.items = [
+      { title: 'Checking', subtitle: '********1234', value: '2215.13' },
+      { title: 'Home Savings', subtitle: '********4234', value: '8676.88' },
+      { title: 'Car savings', subtitle: '********9012', value: '900.13' },
+    ];
+    component.visibleItemsSize = 3;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should show 3 items in list', () => {
```

Comment:
Split into 3 tests for:

1) check size of list
2) check presence of list item
3) check visibility of see all button